### ISSUE TITLE
Strokes: Unhandled Exception Caught: Exception caught

### ISSUE THREAD

**Issue by FirstMM** [CGCookie/retopoflow#922]
**PLEASE READ THE FOLLOWING AND USE THE TEMPLATE TO POST YOUR ISSUE**

*Questions not directly related to the RetopoFlow project are considered general and should be posted elsewhere (e.g. git, blender, python, etc).*

**Before raising the issue, please check the following first:**

- [ ] Read through the README.md file in the branch used to assure the process taken is correct
- [ ] Check the existing [issues](https://github.com/CGCookie/retopoflow/issues) and [pull requests](https://github.com/CGCookie/retopoflow/pulls) to make sure the issue has not already been reported and/or fixed.
- [ ] You can replicate the bug and will provide as much info as possible with blend files, screenshots, logfiles, as well as show the expected result and actual result.

Please prefix your issue name with one of the following: **[BUG]** **[PROPOSAL]** **[QUESTION]**

Issue Template
-----------------------------------

**Issue:**

{*enter your issue here*}

{*add any screenshots of the issue here*}

**How to Reproduce:**

- {*add your list replication steps*}

{*attach any files needed to replicate the issue*}

Automatically Generated Info
-----------------------------------

```
Environment:

- RetopoFlow: 3.0.1
- CG Cookie built
- Blender: 2.91.0 master 2020-11-25
- Platform: Windows, 10, 10.0.19041, AMD64, AMD64 Family 23 Model 113 Stepping 0, AuthenticAMD
- GPU: NVIDIA Corporation, GeForce RTX 2070 SUPER/PCIe/SSE2, 4.5.0 NVIDIA 460.79, 4.50 NVIDIA
- Timestamp: 2020-12-15 21:03:45.146647
- Undo: extrude strip, move grabbed, select, move grabbed, select, move grabbed, select, move grabbed, move grabbed, select

Error Hash: 2fa55a0df9536c56880d357e0a896895

Trace:

- EXCEPTION (<class 'ReferenceError'>): BMesh data of type BMEdge has been removed
-     C:\Users\First\AppData\Roaming\Blender Foundation\Blender\2.91\scripts\addons\RetopoFlow\retopoflow\rfmesh\rfmesh_wrapper.py
- 000 0307:verts() bmv0, bmv1 = self.bmelem.verts
-     C:\Users\First\AppData\Roaming\Blender Foundation\Blender\2.91\scripts\addons\RetopoFlow\retopoflow\rftool_strokes\strokes.py
- 001 0800:extrude_strip() side_verts = list(edges1[0].verts)
-     C:\Users\First\AppData\Roaming\Blender Foundation\Blender\2.91\scripts\addons\RetopoFlow\retopoflow\rftool.py
- 002 0141:wrapper() ret = fn(*args, **kwargs)
-     C:\Users\First\AppData\Roaming\Blender Foundation\Blender\2.91\scripts\addons\RetopoFlow\retopoflow\rftool_strokes\strokes.py
- 003 0356:stroke() if self.replay: self.replay()
-     C:\Users\First\AppData\Roaming\Blender Foundation\Blender\2.91\scripts\addons\RetopoFlow\retopoflow\rfwidget.py
- 004 0108:_callback_tool() fn(self.rftool, *args, **kwargs)
- 005 0128:callback_actions() self._callback_tool('action', *args, **kwargs)
-     C:\Users\First\AppData\Roaming\Blender Foundation\Blender\2.91\scripts\addons\RetopoFlow\retopoflow\rfwidgets\rfwidget_brushstroke.py
- 006 0087:modal_line() self.callback_actions()
-     .../common\fsm.py
- 007 0054:run() return fn(*args, **kwargs)
```

**Comment by vxlcoder** [comment]
Thanks for the report, @FirstMM!

I'm not sure how to reproduce the problem, but I think I know what's causing it and I believe I have a fix for it with commit bb6874b.  If you can try it and let me know, that would be helpful.  otherwise, it will be included in the next minor release.

**Comment by jlampel** [comment]
I believe this is solved. If you have the problem again, please let us know!
